bindings/python/iota_sdk/types/block/block.py

```diff
@@ -39,29 +36,17 @@ class Block:
     slot_commitment_id: HexStr
     latest_finalized_slot: SlotIndex
     issuer_id: HexStr
-    body: BlockBody
-    signature: Signature
-
-    def id(self, params: ProtocolParameters) -> HexStr:
-        """Returns the block ID as a hexadecimal string.
-        """
-        return Utils.block_id(self, params)
 
 
 @json
 @dataclass
 class UnsignedBlock:
-    """An unsigned block type that can hold either a `BasicBlock` or a `ValidationBlock`.
-    Shared data is stored alongside such a block in the header fields.
+    """An unsigned block type that can hold either a `BasicBlockBody` or a `ValidationBlockBody`.
+    Data that is shared between different block body types is stored in the block header.
 
     Attributes:
-        protocol_version: Protocol version of the network to which this block belongs.
-        network_id: The identifier of the network to which this block belongs.
-        issuing_time: The time at which the block was issued. It is a Unix timestamp in nanoseconds.
-        slot_commitment_id: The identifier of the slot to which this block commits.
-        latest_finalized_slot: The slot index of the latest finalized slot.
-        issuer_id: The identifier of the account that issued this block.
-        block: Holds either a `BasicBlock` or a `ValidationBlock`.
+        header: The block header.
```

Comment:
You didn't actually put an header attribute

Reply:
:see_no_evil: